### ISSUE TITLE
Removes spaces from logic inside Angular directives

### ISSUE THREAD

**Issue by gethinoakes** [mohd-akram/vscode-html-format#1]
Just tried this extension out after seeing you posted it on the prettier repo but I've found this issue (albeit a small one)

```
<ng-container *ngIf="formType === 'signup'">
```

becomes
```
<ng-container *ngIf="formType=== 'signup'">
```

Otherwise great, though is it possible to make it so that stuff within an element (e.g. button) gets put onto a new line indented within that element? Instead of just wrapping to the line length rule.

e.g.
```
<button class="btn-login" type="submit" 
[disabled]="!loginForm.valid">login</button>
```

would become
```
<button class="btn-login" type="submit" [disabled]="!loginForm.valid">
   login
</button>
```

thanks.

**Comment by mohd-akram** [comment]
The first case should be fixed. As for the second, the before and after do not represent the same HTML as you added spaces around the content ("login" -> " login "). The formatter does not add spaces/newline if there wasn't one before.

**Comment by mohd-akram** [comment]
This is fixed in version v0.0.3. Thanks for reporting!
